base/static/base/js/jquery.track-everything.js

```diff
@@ -253,19 +276,37 @@
         }
     }
 
-    // Function to handle link clicks and send events to GA4.
-    function handleLinkClick(event, isMiddleClick = false) {
-        const target = event.target.closest('a, button, input');
-        const eventName = getEventName(target);
-        if (!eventName) { return; }
+    function count_indecision(ep, eventName, target) {
+        // Function to count indecision clicks on tabs and dropdowns
+        // --- indecision count logic ---
+        if (eventName === 'tab' || eventName === 'dropdown') {
+            const component = target.closest('[data-ga-category]');
+            if (component) {
+                let count = parseInt(component.getAttribute('data-ga-indecision-count') || '0', 10) + 1;

```

Comment:
If we put `if (isNaN(count)) count = 1;` here, we'll also handle for `NaN` values for cases when data might be corrupted.

```
  function count_indecision(ep, eventName, target) {
      // Function to count indecision clicks on tabs and dropdowns
      // --- indecision count logic ---
      if (eventName === 'tab' || eventName === 'dropdown') {
          const component = target.closest('[data-ga-category]');
          if (component) {
              let count = parseInt(component.getAttribute('data-ga-indecision-count') || '0', 10) + 1;
              if (isNaN(count)) count = 1; // <-- Just add this line
              component.setAttribute('data-ga-indecision-count', count);
          }
      }
  }
```